### PR TITLE
--caveat split issue with caveatname and json

### PR DESCRIPTION
**Pull request by poklu**
https://github.com/authzed/zed/issues/189

### CODE REVIEW

**Comment by poklu** [comment]
I have read the CLA Document and I hereby sign the CLA

**Review by vroldanbet — COMMENTED** [review]
hi! thanks for submitting a PR! would you mind creating a test that reproduces the problem? 🙏🏻

File: internal/commands/relationship.go [hunk 1]
```diff
@@ -264,7 +264,7 @@ func writeRelationshipCmdFunc(operation v1.RelationshipUpdate_Operation) func(cm
 		if operation != v1.RelationshipUpdate_OPERATION_DELETE {
 			caveatString := cobrautil.MustGetString(cmd, "caveat")
 			if caveatString != "" {
-				parts := strings.Split(caveatString, ":")
+				parts := strings.SplitN(caveatString, ":",2)
```
Comment (vroldanbet) [inline internal/commands/relationship.go:267]: so what this solves is the scenario where your caveat value also includes a colon? Mind sharing your example?

**Review by josephschorr — APPROVED** [review]
(looks like lint broke)

**Review by josephschorr — CHANGES_REQUESTED** [review]

**Review by josephschorr — CHANGES_REQUESTED** [review]
LGTM

**Review by josephschorr — APPROVED** [review]

**Pull request merged**